README.md

```diff
@@ -24,12 +24,20 @@ pip install beautifulsoup4
 
 ```--time_delay``` : If set to True, there will be a delay of 10 seconds between consecutive url requests (recommended). Default = ```True```
 
+```--spread```: If passed, number of articles collected is spread across all categories passed in. If `most_popular` in categories, all its articles are collected and the remainder is spread across other categories. 
```

Comment:
can you add this description, especially the part about the `most popular` category, to spread's argparse description in scraper.py